scripts/sbin/start-cli.sh

```diff
@@ -61,7 +61,7 @@ fi
 
 # Added parameters when default parameters are missing
 user_param="-u root"
-passwd_param="-pw root"
+passwd_param="-pw IoTDB@2017"
```

Comment:
Just come up a question. If upgrading from an old version, the password is still `root`. Will using this CLI without input password meet error?

Reply:
Yes, that cannot be avoided. 
The user should modify the scripts manually or avoid upgrading the scripts.